products/jse_sub/features/accessibility.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Verifies that the Accessibility text appears
```

Comment:
The reason why we didn't put this in all sites was because different products have their accessibility text in different pages.